PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/appservice/action/SSHTerminalManager.java

```diff
@@ -128,6 +59,10 @@ public void openConnectionInTerminal(ShellTerminalWidget shellTerminalWidget, Cr
             while ((shellTerminalWidget.getTtyConnector() == null) && count++ < 200) {
                 Thread.sleep(100);
             }
+
+            while ((shellTerminalWidget.getTerminalStarter() == null) && count++ < 200) {
+                Thread.sleep(100);
+            }
```

Comment:
what it the purpose to add these codes? they are duplicated

Reply:
bty, As the Proxy is implemented by us, is there any way to provide a connection-completion notification/interface in it? 
so that we can avoid the `while sleep` try codes.

Reply:
getTerminalStarter may be null